frontend/src/scenes/cdp/ExportOverview.tsx

```diff
@@ -0,0 +1,277 @@
+import { useActions, useValues } from 'kea'
+import { LemonTable } from 'lib/lemon-ui/LemonTable'
+import { LemonTag, LemonTagPropsType } from 'lib/lemon-ui/LemonTag/LemonTag'
+import { LemonButton } from 'lib/lemon-ui/LemonButton'
+import { ChangeExportRunStatusEnum, BatchExportRunType } from './types'
+import { TZLabel } from 'lib/components/TZLabel'
+import { ProfilePicture } from 'lib/lemon-ui/ProfilePicture'
+import { More } from 'lib/lemon-ui/LemonButton/More'
+import { LemonDivider } from '@posthog/lemon-ui'
+import { LemonDialog } from 'lib/lemon-ui/LemonDialog'
+import { Progress } from 'antd'
+import { ExportRun } from './ExportRun'
+import { BatchExportLogic } from './BatchExportLogic'
+
+export enum BatchExportStatus {
+    Running = 'Running',
+    Cancelled = 'Cancelled',
+    Completed = 'Completed',
+    ContinuedAsNew = 'ContinuedAsNew',
+    Failed = 'Failed',
+    Terminated = 'Terminated',
+    TimedOut = 'TimedOut',
+    Starting = 'Starting',
+    Paused = 'Paused',
+}
+
+export function StatusToTagType(status: BatchExportStatus): LemonTagPropsType {
+    switch (status) {
+        case BatchExportStatus.Running:
+            return 'highlight'
+        case BatchExportStatus.Cancelled:
+            return 'warning'
+        case BatchExportStatus.Completed:
+            return 'success'
+        case BatchExportStatus.ContinuedAsNew:
+            return 'highlight' // TODO: understand what this does and choose a more appropriate color
```

Comment:
In a few words, `Continued-As-New` means a run was retried. In more words: A new instance of a workflow (a workflow run) is created by Temporal with the same state as an existing instance. This new workflow run has it's own, different id and history.